tests/test_smoke.py

```diff
@@ -156,6 +158,26 @@ def test_minimal():
     run_one_test(test)
 
 
+@pytest.mark.xdist_group(name="serial_lambda_labs")
```

Comment:
nit: would be nice to have a comment above the decorator explaining the reason we need to run them sequentially.

Reply:
Good idea.

Reply:
Fixed!